VERSION

```diff
@@ -1 +1 @@
-0.45.0
+0.45.1
```

Comment:
see above, the version will be adjusted once we cut a release.

Reply:
@s-urbaniak should the go.mod change to the patch version be rolled back as well?